commands/misc.js

```diff
@@ -23,7 +23,7 @@ exports.commands = {
 	randomanswer: function (arg, user, room) {
 		if (room === user) return false;
 		var text = '';
-		var rand = ~~(20 * Math.random());
+		var rand = ~~(23 * Math.random());
```

Comment:
More like:
```js
var text = '';
var answers = this.trad('answers');
var rand = ~~(answers.length * Math.random());
```

With this solution, it should be no problem, if different languages have different amounts of answers.

Tagging @Ecuacion for a opinion on this case.